packages/paste-website/src/pages/components/side-modal/index.mdx

```diff
@@ -292,8 +292,9 @@ The SideModalButton renders a Button component and accepts all of its props, whi
 | Prop         | Type              | Description                                                                               | Default        |
 | ------------ | ----------------- | ----------------------------------------------------------------------------------------- | -------------- |
 | `aria-label` | `string`          | Title of the dialog for screen readers.                                                   |                |
+| `hideOnEsc?` | `boolean`         | Set to `true` to enable closing the SideModal when the "Escape" key is pressed.           | false          |
```

Comment:
Issue: just need to reflect the new default value here, and maybe tweak the change set text